pyodata/v2/service.py

```diff
@@ -235,11 +235,11 @@ def __repr__(self):
 class ODataHttpRequest:
     """Deferred HTTP Request"""
 
-    def __init__(self, url, connection, handler, headers=None):
+    def __init__(self, url, connection, handler, headers={}):
```

Comment:
Please, don't use objects for default values. The default value is instantiated only once. In this case, once someone populates headers, all other consecutive calls of ODataHttpRequests() will receive the hedaers directory populate.

Or was that your intention?

Reply:
That was my intention for my use case, a bit hacky but works well for us without needing to set global params. 

However, in retrospect you are correct we should not be forcing the headers on all descendant classes as this is definitely not expected behaviour. 

I will revise this to use the dict type as the default value.